files/galaxy/config/pawsey_object_store_conf.xml

```diff
@@ -1,23 +1,8 @@
 <?xml version="1.0"?>
 <object_store type="hierarchical">
     <backends>
-        <backend id="perthNFS4" type="disk" order="0">
-            <files_dir path="/mnt/user-data-4" />
-        </backend>
-        <backend id="perthNFS2" type="disk" order="1">
-            <files_dir path="/mnt/user-data-2" />
-        </backend>
-        <backend id="perthNFS3" type="disk" order="2">
-            <files_dir path="/mnt/user-data-3" />
-        </backend>
-        <backend id="perthNFS1" type="disk" order="3">
-            <files_dir path="/mnt/user-data" />
-        </backend>
-        <backend id="GPFS" type="disk" order="4">
-            <files_dir path="/mnt/galaxy/files" />
-        </backend>
-        <backend id="NFS" type="disk" order="5">
-            <files_dir path="/mnt/files2" />
+        <backend id="loadtest" type="disk" order="0">
```

Comment:
we need to keep all the other object stores as well, so we can access the input data for the workflows during the tests. This data is already on the server in one of the object stores.

Reply:
Oops, I forgot about this.  So this needs to have the new entry (/mnt/user-data/load-test) with index "0" and all of the existing entries with index "1" to "7" instead of deleting them.